tests/core.js

```diff
@@ -424,20 +432,29 @@ require([
 
         describe("drag+drop should", function () {
             var mugs;
-            before(function () {
-                util.loadXML(INSERT_QUESTIONS_XML);
-                util.addQuestion.bind({prevId: "text2"})("Repeat", "repeat");
-                mugs = {
-                    text1: call("getMugByPath", "/data/text1"),
-                    text2: call("getMugByPath", "/data/text2"),
-                    repeat: call("getMugByPath", "/data/repeat"),
-                    group: call("getMugByPath", "/data/group"),
-                    text3: call("getMugByPath", "/data/group/text3"),
-                    text4: call("getMugByPath", "/data/group/text4"),
-                    hidden2: call("getMugByPath", "/data/group/hidden2"),
-                    text5: call("getMugByPath", "/data/text5"),
-                    hidden1: call("getMugByPath", "/data/hidden1")
-                };
+
+            before(function (done) {
+                util.init({
+                    features: {'experimental_ui': false},
+                    core: {
```

Comment:
This change doesn't seem necessary since the `describe` is nested in another `describe` that already has vellum initialized with experimental_ui turned off.